tests/experimental/pydantic/test_conversion.py

```diff
@@ -946,32 +895,6 @@ class User(BaseModel):
             password: Optional[str] = Field(default=None, default_factory=lambda: None)
 
 
-def test_can_convert_pydantic_type_to_strawberry_with_additional_field_resolvers():
```

Comment:
This test is not a valid strawberry behavior. strawberry doesn't create dataclasses fields for fields with resolvers. so I removed this for now. let me know what you think.

Reply:
@patrick91 what do you think?